tools/cli/commands.js

```diff
@@ -375,7 +375,7 @@ function doRunCommand(options) {
     oplogUrl: process.env.MONGO_OPLOG_URL,
     mobileServerUrl: utils.formatUrl(parsedMobileServerUrl),
     once: options.once,
-    noReleaseCheck: options['no-release-check'],
+    noReleaseCheck: options['no-release-check'] || process.env.METEOR_NO_RELEASE_CHECK,
```

Comment:
I'm not sure this deserves of an environment variable.   Why wouldn't this just be a `--flag`?

(I realize that Meteor has a lot of other ENV variables and it's a common way to set things, however I think ENVs should be reserved for production environments.  This particular flag wouldn't have any significance in production).